src/main/java/ca/ciccc/BankAccount.java

```diff
@@ -100,6 +102,16 @@ public void withdraw(double amount) {
     public boolean transactionFee(double fee) {
         // TODO 2: Your code goes here.
 
+        int transactions;
+        double amount = 0.0;
+
+        for(transactions = 0; transactions <= transactionCount; transactions++) {
+            amount += (fee * transactions);
+        }
+
+        if (balance >= amount) {
+            return true;
+        }
         return false;
```

Comment:
[FYI] Instead of using `if` statement, you can also return a boolean value like the following:

```suggestion
        return balance >= amount;
```